{{cookiecutter.github_repository}}/provisioner/ansible.cfg

```diff
@@ -0,0 +1,2 @@
+[defaults]
+allow_world_readable_tmpfiles = True
```

Comment:
This can directly go master with ansible.cfg, no need to wait for this pr to be merged.